### DISCUSSION TITLE
Installation hangs at cursor and blank screen after initial installation type selection menu

### DISCUSSION THREAD

**Discussion by tsmith-spscc** [Security-Onion-Solutions/securityonion#13329] [2.4] [answered]
### Version

2.4.80

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

16

### RAM

256 GB

### Storage for /

20 TB

### Storage for /nsm

20 TB

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I cannot for the life of me get SO 2.4.80 to install on a new physical server. No matter what I try, the installer hangs at a cursor with a blank screen after I select the initial type of SO install I want to perform (standard, desktop, IDH, basic graphics mode, etc). I've tried both the standard and basic graphics modes multiple times to no avail. I've also tried two different USBs using the 2.4.80 ISO burned via Balena Etcher - both have the same problem. I've tried both UEFI and Legacy modes, same issue. 

Some investigation into this issue seems to indicate that perhaps it has something to do with a display driver? The advice given here: https://docs.securityonion.net/en/2.4/trouble-booting.html regarding the "nomodeset" option appears to be set by choosing the basic graphics install option. Even still, I've attempted to set that option manually to no effect.

The last set of text I see before the installer fails out at the cursor of death is the following (truncated to some degree):

* If the graphical installation interface fails to start, try again with the inst.text bootoption to start text installation

I have tried adding inst.text to the boot options using "e" when at the initial SO installation selection menu and of course this doesn't work either. Perhaps I'm doing it incorrectly, as I've not found a lot of good information on how to properly add it to the command string, but who knows. 

I've successfully installed the previous version, 2.4.70, on the exact same type of hardware (another server from the same vendor with the exact same specs), so I have no idea why this one won't work. Any suggestions are greatly appreciated. Thanks.

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by dougburks** [comment]
When you downloaded the ISO image, did you verify it as shown at https://securityonion.com/download?

**Reply by tsmith-spscc** [reply]
Yes. It verified fine.

**Reply by dougburks** [reply]
This is a strange issue. Our standard installation is a text mode installer so it shouldn't really be an issue with display drivers. Also, I'm not aware of any changes between 2.4.70 and 2.4.80 that would cause something like this.

Here are some things you can try:
- try a different USB port on the server
- make sure that Balena Etcher is set to verify the USB drive after flashing
- try a 3rd USB drive from a different manufacturer
- try a different HDMI cable
- try a different monitor
- try an Oracle Linux 9 ISO image to see if that behaves differently
- run full hardware diagnostics on the hardware

**Reply by tsmith-spscc** [reply]
Apologies for the late reply. I will be working on this again tomorrow. I will try your suggestions and post the results either way. Thanks.

**Reply by tsmith-spscc** [reply, marked as answer]
So the issue was with my RAID array. I had to rebuild it. Once I had that taken care of, I was able to install Security Onion with no issue.
